go.mod

```
@@ -1,8 +1,6 @@
 module github.com/dev-hato/tfrbac
 
-go 1.21
-
-toolchain go1.23.6
+go 1.23.6
```

```suggestion
go 1.23

toolchain go1.23.6
```

みたいに `go` ではマイナーバージョンまで、 `toolchain` ではパッチバージョンまで指定できると思ってたけど、そうでもない説？

---

https://zenn.dev/haruyama480/articles/7e3b00bfad69ae?utm_source=chatgpt.com#%E8%A7%A3%E6%B1%BA%E6%96%B9%E6%B3%95-(%E5%85%BC%E3%83%99%E3%82%B9%E3%83%88%E3%83%97%E3%83%A9%E3%82%AF%E3%83%86%E3%82%A3%E3%82%B9)

これはこれでいいはず

というかgoコマンドで上げたから、これで正しい
```
rm go.mod go.sum
go mod init
go mod tidy
```

---

ヘェ〜。